### ISSUE TITLE
README asset

### ISSUE THREAD

**Issue by jcwleo** [jcwleo/random_network_distilation_pytorch#5]
![image](https://user-images.githubusercontent.com/23333028/48664090-cf836680-eadc-11e8-969b-5201db99907d.png)

**Comment by jcwleo** [comment]
![image](https://user-images.githubusercontent.com/23333028/50719328-de9a9400-10dd-11e9-8c8c-29f7709cdf1d.png)

**Comment by kslazarev** [comment]
https://github.com/jcwleo/random-network-distillation-pytorch/blob/master/config.conf
Is this config the last one to get similar results as on images above?

I see last pull request is about normalization, maybe UseNorm = True improve reward_per_epi or speed of convergence? And what about UseNoisyNet, when it could better to use?

**Comment by kslazarev** [comment]
Result by config in master branch. 

MontezumaRevengeNoFrameskip-v4
<img width="1064" alt="2019-01-11 15 05 01 gmt 03 00" src="https://user-images.githubusercontent.com/371191/51032926-bd8ce400-15b2-11e9-96ec-500eb7ea5263.png">

Right now, set NumEnv==128 and MaxStepPerEpisode==4500.
I'll attach result when it's get 1200-2000 steps.

**Comment by kslazarev** [comment]
@jcwleo I see the difference in x-axis scale in reward_per_epi and reward_per_rollout plots.
On your MontezumaRevengeNoFrameskip-v4 image they are 1.200k and 12.00k (10x scale).
But on my temporary progress image they are 200 and 600 (3x scale). Maybe need to change additional option in config?
<img width="1068" alt="2019-01-11 21 58 49 gmt 03 00" src="https://user-images.githubusercontent.com/371191/51053935-378d8f00-15ec-11e9-8d94-f50a1aaab39e.png">

**Comment by kslazarev** [comment]
Or the x-axis scale (global_update and sample_episode) depends on player survival so on later updates x-axis scale will be the same?

**Comment by jcwleo** [comment]
@kslazarev per_rollout and per_epi is not same scale. per_rollout means just one global update(enter agent.train_model()). but per_epi means Env’s one episode info that is one of parallel env.
If one episode’s total step is 1024 and Num_step(rollout size) is 128, each scale of x-axis is 8 times different.

**Comment by kslazarev** [comment]
@jcwleo Yes, correct. I have another small questions about code. How could be appropriate to ask? Every question as issue, or move forward to ask in this issue?

**Comment by jcwleo** [comment]
@kslazarev I want you to create an issue for each question. :)

**Comment by kslazarev** [comment]
NumEnv=128 and MaxStepPerEpisode==4500
<img width="1065" alt="2019-01-13 7 40 16 gmt 03 00" src="https://user-images.githubusercontent.com/371191/51085782-1ee9ba00-174f-11e9-8b6d-5695ace4a706.png">
NumEnv=16 and MaxStepPerEpisode==4500
<img width="1066" alt="2019-01-13 16 20 57 gmt 03 00" src="https://user-images.githubusercontent.com/371191/51085801-46408700-174f-11e9-9980-f34a995d8980.png">

Looks similar as in README. On NumEnv=128 I've stopped process because swap memory are used.

**Comment by xiaioding** [comment]
Hello, can you tell me how many Gpus you used and how long it took you to see this effect?

**Comment by kslazarev** [comment]
Hello. Not fast. Don't remember exactly, 1 or 2 NV 1080 Ti

**Comment by xiaioding** [comment]
@kslazarev Excuse me, I use 1 3090,2 envs, run for more than 2 hours, the reward is still 0, is this normal? I didn't load the pre-training model

**Comment by kslazarev** [comment]
It was 3 years ago. Could not help, I don't remember exactly what problem could cause.

**Comment by xiaioding** [comment]
@kslazarev Ok, thanks

**Comment by jcwleo** [comment]
@kslazarev 
Thank you for answering for me.
